### PR TITLE
Define a SyntaxError

### PR DESCRIPTION
**Pull request by jonasfj**
This is a PR for your PR... :)

### CODE REVIEW

**Comment by jonasfj** [comment]
Just because it took me forever to find out how to make custom errors last time I had to do it..
This error things is a very tiny dependency, and it gets the stuff with stack traces right... I guess we don't really need those, but it's nice to have anyways, so you know where to look if you're debugging it..

File: src/tokenizer.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
-/*
-* Author: Jonas Finnemann Jensen
-* Github: https://github.com/jonasfj
-*/
+import assert from 'assert';
+import ExtendableError from 'es6-error';
```
Comment (hammad13060) [inline src/tokenizer.js]: I don't think es6 supports import statements ?? Are you using some other preset ??

**Comment by jonasfj** [comment]
The taskcluster preset :)

Den 3. feb. 2017 8.45 AM skrev "Hammad Akhtar" <notifications@github.com>:

**Comment by hammad13060** [comment]
I am using this preset ```babel-preset-taskcluster```. Is this fine ??

**Comment by jonasfj** [comment]
Yeah that's the one... It should add await/async and import...

Den 3. feb. 2017 8.48 AM skrev "Hammad Akhtar" <notifications@github.com>:

**Comment by jonasfj** [comment]
> This looks good. I will cherry pick this once I resolve all the other issues.
You don't have to cherrypick, you can either just copy/paste and close the PR... or merge it... the big merge button :)

**Comment by jonasfj** [comment]
Looks like it made into the other PR :)

**Pull request closed without merging**